.tekton/konflux-testing-build.yaml

```diff
@@ -9,7 +9,9 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
     # TODO(ROX-21073): re-enable for all PR branches
-    pipelinesascode.tekton.dev/on-cel-expression: (event == "push" && target_branch == "master") || (event == "pull_request" && (source_branch.contains("rhtap") || source_branch.contains("konflux")))
```

Comment:
The code in this test bed repo is outdated, this is the current CEL expression for comparison: 

```
pipelinesascode.tekton.dev/on-cel-expression: |
  (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
  (event == "pull_request" && (source_branch.matches("(konflux|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
```

The diff is 

```
- (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
+ (event == "push" && target_branch.matches("refs/tags/*|^(master|release-.*)$")) ||
```

Added `refs/tags/*|` to the `target_branch` regex for `push` events.

Source: https://github.com/stackrox/stackrox/blob/master/.tekton/central-db-build.yaml#L12-L14